### ISSUE TITLE
@types/request duplicate identifier 'time'

### ISSUE THREAD

**Issue by walexnelson** [DefinitelyTyped/DefinitelyTyped#18074]
- [x] I tried using the `@types/request` package and had problems.

Latest release seems to have added a duplicate 'time' identifier to the CoreOptions interface

```
node_modules/@types/request/index.d.ts(146,9): error TS2300: Duplicate identifier 'time'.
node_modules/@types/request/index.d.ts(157,9): error TS2300: Duplicate identifier 'time'.
```

- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @soywiz @Bartvds @joeskeen @ccurrens

**Comment by JonathanYates** [comment]
I'm getting the same.

How on earth do these things end up getting released. Clearly broken and braking my build!!

Please fix immediately.

**Comment by scttcper** [comment]
#18077

**Comment by c4d3r** [comment]
A temporary fix can be adding an older version of the package `"@types/request": "0.0.42"` worked for me (add this as well if you are using `request-promise`)

**Comment by underscorenico** [comment]
The same here, please fix

**Comment by dflor003** [comment]
Breaking the build in a few projects for me too. Please fix this!

**Comment by richrout** [comment]
Appears to be fixed with latest commit (just published v2.0)
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/40e971e511afb76e0d808a47198efde512bba1b8

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
